setup.py

```diff
@@ -45,6 +45,7 @@
         'opencensus-context >= 0.2.dev0',
         'google-api-core >= 1.0.0, < 2.0.0; python_version<"3.6"',
         'google-api-core >= 1.0.0, < 3.0.0; python_version>="3.6"',
+        "six ~= 1.16.0",
```

Comment:
```suggestion
        "six ~= 1.16",
```

I imagine any major version 1 should be OK